tests/unit/components/setup.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * The tests for our components need React, because we are using the new
+ * technology of shallow rendering, so we are passing to the renderer
```

Comment:
The reason we need `React` in tests is not because of shallow rendering, it's that we use JSX.